x-pack/plugins/apm/public/context/url_params_context/url_params_context.test.tsx

```diff
@@ -52,8 +52,8 @@ describe('UrlParamsContext', () => {
     const params = getDataFromOutput(wrapper);
 
     expect([params.start, params.end]).toEqual([
-      '2010-03-15T00:00:00.000Z',
-      '2010-04-11T00:00:00.000Z',
+      '2010-03-15T12:00:00.000Z',
+      '2010-04-10T12:00:00.000Z',
```

Comment:
I don't understand the previous values. How did `2010-03-15T12:00:00Z` ever get rounded to `2010-03-15T00:00:00.000Z`?

Reply:
I believe that was what the previous rounding outputted 

```js
const ticks = scaleUtc().domain([start, end]).nice().ticks();
```